src/Commands/ToggleDone.ts

```diff
@@ -61,7 +61,7 @@ export const toggleDone = (checking: boolean, editor: Editor, view: MarkdownView
  *
  * @interface EditorInsertion
  */
-interface EditorInsertion {
+export interface EditorInsertion {
```

Comment:
We are unsure whether is interface is a good idea to show to API users.

Do you find the `moveTo` field useful? Or you are just returning it because it is in the correct code?

If all you need is the `text` field we would suggest returning `EditorInsertion.text` from your function and not exposing the interface.

Reply:
Yes, just using `text` will do for my use case.